fibbingnode/misc/mininetlib/iprouter.py

```diff
@@ -98,6 +99,29 @@ def __init__(self, router, *args, **kwargs):
         self.ospf.redistribute.static = 1000
         self.ospf.router_id = router.id
 
+        # Parse LSA throttling parameters
+        delay = CFG.get("DEFAULT", 'delay')
+        delay = delay if delay != '' else 0
```

Comment:
Putting the defaults in the res/default.cfg ensures that it exists, as it gets automatically overwritten in https://github.com/Fibbing/FibbingNode/blob/master/fibbingnode/southbound/main.py#L174 if the user specifies an alternate config